source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -612,6 +634,66 @@ export default class NetworkStatusStore extends Store {
     this.stateDirectoryPath = stateDirectoryPath;
   };
 
+  @action _logStateSnapshot = () => {
```

Comment:
@DeeJayElly please add a comment above this line with an explanation of what this method does 🙏 
PS. Why is this method defined as mobx action? AFAIK it does not mutate the state...

Reply:
Done